### PR TITLE
Abb 161 and others

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Abb is a composite manuscript and has the pagination in two directions (2+)1-264 and 1-94(+2), and is
 quite difficult to handle. I encode `<locus/>` basing on my thorough pagination which I continued into Unit II, 
I wonder if it is ok like this).

### CODE REVIEW

File: ParisBNF/abb/BNFabb161.xml [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+                    <summary/>
+                     <msItem xml:id="p2_i1">                      
+                        <locus from="133r" to="181r">2-94</locus>
+                       <title type="complete" ref="LIT2077Nuzhat"/>
+                        <textLang mainLang="ar"/>
+                       <colophon xml:lang="ar" xml:id="coloph2"/>                       
+                                       
+                        <note>Paginated in the reversed direction, from right to left. The colophon indicates that
+                          the copying was completed in the time from 9 December 1847 to 7 January 1848, 
+                          for <persName ref="PRS1072dAbbadi">Michel d’Abbadie</persName>. Recorded falsly in
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb161.xml:241]: fals**e**ly

File: ParisBNF/abb/BNFabb161.xml [hunk 2]
```diff
@@ -0,0 +1,370 @@
...
+                  <msIdentifier/>
+                  <msContents>
+                    <summary/>
+                     <msItem xml:id="p2_i1">                      
+                        <locus from="133r" to="181r">2-94</locus>
+                       <title type="complete" ref="LIT2077Nuzhat"/>
+                        <textLang mainLang="ar"/>
+                       <colophon xml:lang="ar" xml:id="coloph2"/>                       
+                                       
+                        <note>Paginated in the reversed direction, from right to left. The colophon indicates that
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb161.xml:239]: I would say
Paginated from right to left

File: ParisBNF/abb/BNFabb161.xml [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+                            <width>165</width>
+                          </dimensions>
+                        </extent>               
+                      
+                        <foliation>The manuscript is paginated. The first unpaginated leaf is assigned page numbers <locus target="#i #ii"/>.
+                          Only on <locus from="1r" to="2r">1-5</locus> and <locus target="#132">263-264</locus> 
+                          numbers are written on all pages, 
+                          in-between (the biggest part of the volume) the numbers are written only on recto-sides of the leaves. 
+                          The part accomodating <ref  target="#ms_i4"/> in Arabic is paginated in the opposite direction, from right to left,
+                          and the page numbers (from 1 to 94) are written only on the verso-sides of the leaves. The last leaf of the volume is not paginated
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb161.xml:54]: from 2 to 94?
Number 1 cannot possibly be written on a verso side, in continuous pagination recto is only for even numbers (2-4-6-8 etc)

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ParisBNF/abb/BNFabb161.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb161.xml:241]: ok

File: ParisBNF/abb/BNFabb161.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb161.xml:239]: ok

File: ParisBNF/abb/BNFabb161.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb161.xml:54]: But it is on the verso side. Pagination of the Arabic part goes on the verso-sides, 1- 3-5-7... 93-(94 supplied by me, on the recto). The next number on the next verso is 264, and it switches to the recto and goes further again with odd numbers. It is quite confusing.

Comment (eu-genia) [reply ParisBNF/abb/BNFabb161.xml:54]: No. For Arabic, just like for all others, the recto is the first page and verso is the second. The orientation is different. If you have 1-3-5-7 etc it is the pagination on recto. Then it is from 1 to 93.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**